CRT.c

```diff
@@ -74,6 +74,22 @@ const char *const *CRT_treeStr = CRT_treeStrAscii;
 
 int CRT_delay;
 
+const char* CRT_degreeSign;
+
+static const char* initDegreeSign(void) {
+#ifdef HAVE_LIBNCURSESW
+   if (CRT_utf8)
+      return "\xc2\xb0";
+#endif
+
+   static char buffer[4];
+   int r = snprintf(buffer, sizeof(buffer), "%lc", 176);
```

Comment:
no (maybe i'll add a comment): this might actually fail on platforms not supporting wide characters, e.g. with POSIX locale (occurred on a raspberry pi)

Reply:
I feared something like that. ;-) Comment will be appreciated.